pages/blog/_posts/2020-01-28-Thanos-infinite-metrics.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: blogpost
+permalink: /blog/metrics-unlimited-thanos/
+title: "Towards Observability nirvana: Infinite metric retention with Thanos"
+date: 2020-01-28
+tags: infrastructure observability prometheus thanos kubernetes
+author: <a href="https://www.linkedin.com/in/hihrig/">Holger Ihrig</a>
+---
+In nowadays DevOps world our industry is relying on the ability to observe and monitorize our infrastructure and 
```

Comment:
In the current DevOps world, our industry relies on

Reply:
Done